spring-context/src/main/java/org/springframework/scheduling/concurrent/ThreadPoolTaskScheduler.java

```diff
@@ -64,6 +64,10 @@
 
 	private volatile boolean removeOnCancelPolicy;
 
+	private volatile boolean executeExistingDelayedTasksAfterShutdownPolicy;
```

Comment:
Default value in `ScheduledThreadPoolExecutor` is `true`. Does it matter if it's different?

Reply:
I believe you are actually correct and it matters for the getter:
if we don't have a scheduled executor inside our class - we return local mentioned value.

Reply:
fyi I added a default true value corresponding to scheduled thread pool executor